src/main/java/graphql/schema/GraphQLAppliedArgument.java

```diff
@@ -0,0 +1,255 @@
+package graphql.schema;
+
+
+import graphql.Assert;
+import graphql.PublicApi;
+import graphql.language.Argument;
+import graphql.language.Value;
+import graphql.util.TraversalControl;
+import graphql.util.TraverserContext;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Consumer;
+
+import static graphql.Assert.assertNotNull;
+import static graphql.Assert.assertValidName;
+import static graphql.execution.ValuesResolver.getInputValueImpl;
+
+/**
+ * This defines an argument and its value that can be supplied to a graphql field (via {@link GraphQLFieldDefinition} during
```

Comment:
I don't think this is a true this is used for field argument values. conceptually this might be true, but a GraphQLSchemaElement is not concerned with queries/